modules/squants/src/main/scala/pureconfig/module/squants.scala

```diff
@@ -0,0 +1,179 @@
+  implicit val luminousFluxConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[LuminousFlux](LuminousFlux.apply, _.toString)
+  implicit val luminousIntensityConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[LuminousIntensity](LuminousIntensity.apply, _.toString)
+
+  // radio
+  implicit val irradianceConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Irradiance](Irradiance.apply, _.toString)
+  implicit val radianceConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Radiance](Radiance.apply, _.toString)
+  implicit val spectralIntensityConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SpectralIntensity](SpectralIntensity.apply, _.toString)
+  implicit val spectralIrradianceConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SpectralIrradiance](SpectralIrradiance.apply, _.toString)
+  implicit val spectralPowerConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SpectralPower](SpectralPower.apply, _.toString)
+
+  // space
+  implicit val angleConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Angle](Angle.apply, _.toString)
+  implicit val areaConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Area](Area.apply, _.toString)
+  implicit val lengthConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Length](Length.apply, _.toString)
+  implicit val solidAngleConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[SolidAngle](SolidAngle.apply, _.toString)
+  implicit val volumeConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Volume](Volume.apply, _.toString)
+
+  // thermal
+  implicit val temperatureConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Temperature](parseTemperature, _.toString)
+  implicit val thermalCapacityConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[ThermalCapacity](ThermalCapacity.apply, _.toString)
+
+  // time
+  implicit val frequencyConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Frequency](Frequency.apply, _.toString)
+  implicit val timeConfigConvert =
+    ConfigConvert.nonEmptyStringConvert[Time](Time.apply, _.toString)
+
+  // This is temporary until https://github.com/typelevel/squants/pull/183 is released (1.2.0 ???)
+  // Without it, ScalaCheck has a pretty easy time breaking the conversion
+  private[squants] def parseTemperature(s: String): Try[Temperature] = {
+    val regex = "([-+]?[0-9]*\\.?[0-9]+(?:[eE][-+]?[0-9]+)?)*°? *(.{1})".r
+    s match {
+      case regex(value, unit) => unit match {
+        case Fahrenheit.symbol | "f" | "F" => Success(Fahrenheit(value.toDouble))
+        case Celsius.symbol | "c" | "C" => Success(Celsius(value.toDouble))
+        case Kelvin.symbol | "k" | "K" => Success(Kelvin(value.toDouble))
+        case Rankine.symbol | "r" | "R" => Success(Rankine(value.toDouble))
+        case unknownUnit => Failure(new Exception(s"PureConfig library programmer error: The regex '$regex' found unit '$unknownUnit' which was not defined."))
```

Comment:
@cranst0n I may have confused you. I was thinking that this error would only occur if regex included a unit character which wasn't supported here. I was hoping this wouldn't be an error that a normal user of the library would see.
The way you've defined things now, an input of 13° will hit this case, reporting ° as an invalid unit. 13(space) will also hit this case.

I think you should go back to a defined set of expected units in the regex, causing this case to only be hit if the two lists of units end up diverging.